### PR TITLE
Cuba focus tree

### PR DESCRIPTION
**Pull request by TheBrit7**

### CODE REVIEW

**Review by marijn211 — COMMENTED** [review]
See error log

File: .gitignore
Comment (ilkober) [inline .gitignore:1]: @marijn2112 check this please to be sure

File: localisation/english/state_names_l_english.yml
Comment (ilkober) [inline localisation/english/state_names_l_english.yml:1]: STATE_314_RENAMING:0 "[314.controller.GetStateName_313]"

 STATE_315:1 "Oriente"
 STATE_315_RENAMING:0 "[313.controller.GetStateName_315]"
 Are wrong

**Review by ilkober — COMMENTED** [review]
A few things to add for @marijn2112  review

**Comment by marijn211** [comment]
I dont know why, but the change .gitignore still doesnt make the descriptor be ignored as Fantom suggested

I think it might only work for files that aren't part of the repository?

**Review by marijn211 — APPROVED** [review]

**Pull request merged**